doc/ReleaseNotes/_ReleaseNotes.md

```diff
@@ -165,7 +166,7 @@
 * Time picker flyout default styles has been changed to include done and cancel buttons
 * DataTemplateSelector implementations are now called using the 2 parameters overload first with a fallback to the 1 parameter overload on null returned value.
   Old behavior could be restored using `FeatureConfiguration.DataTemplateSelector.UseLegacyTemplateSelectorOverload = true`.
-* Using "/n" directly in the XAML for a text/content property is not supported anymore in order to match the UWP behavior. 
+* Using "/n" directly in the XAML for a text/content property is not supported anymore in order to match the UWP behavior.
```

Comment:
A lot of changes here - bad merge?

Reply:
That's definitely a merging problem.

Reply:
Oooh... no that's a not a merge problem... it's because I move the `.editorconfig` and the editor applied the rules on the file.